packages/manager/apps/pci-savings-plan/src/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import {
+  ShellContext,
+  initShellContext,
+  initI18n,
+} from '@ovh-ux/manager-react-shell-client';
+import App from './App';
+import '@ovhcloud/ods-theme-blue-jeans/dist/index.css';
+import './index.scss';
+import './vite-hmr';
+
+import { UNIVERSE, SUB_UNIVERSE, APP_NAME, LEVEL2 } from './tracking.constant';
+
+const trackingContext = {
+  chapter1: UNIVERSE,
+  chapter2: SUB_UNIVERSE,
+  chapter3: APP_NAME,
+  appName: APP_NAME,
+  pageTheme: UNIVERSE,
+  level2Config: LEVEL2,
+};
```

Comment:
**nitpick:** Can you update the variables with the right values?

Reply:
the values `UNIVERS`, `SUB_UNIVERS`, ... are not literals, they are constants defined and initialized in the file `packages/manager/apps/pci-savings-plan/src/tracking.constant.ts`